### PR TITLE
feat: Feature/moduel context

### PR DESCRIPTION
**Pull request by Noggling**
Added module state context to echo framework. and will update documentation plus plsu.

### CODE REVIEW

**Review by ingridklepsvik — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+![logo](https://raw.githubusercontent.com/equinor/EchoFramework/main/doc/ee.png)
+
+# EchoFramework
+
+This is a monorepo for @equinor/echo-framework and @equinor/echo-client. This is where echo-core and base is utilized.
+
+## @equinor/echo-framework
+
+Component library / framework fro echo. this library contains components that are vital to the client.
```
Comment (OveBDev) [inline README.md:9]: . This

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## @equinor/echo-client
+
+This is a light wait client used under module development.
```
Comment (OveBDev) [inline README.md:13]: weight

File: README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+# Speck
+
+The speck documentation fro echo framework and client can be found in the links below.
```
Comment (OveBDev) [inline README.md:17]: spec

File: README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## @equinor/echo-framework
+
+Component library / framework fro echo. this library contains components that are vital to the client.
+
+## @equinor/echo-client
+
+This is a light wait client used under module development.
+
+# Speck
```
Comment (OveBDev) [inline README.md:15]: Spec

File: README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+-   [echo-framework-spec](https://github.com/equinor/EchoFramework/tree/main/doc/echo-framework.md)
+-   [echo-client-spec](https://github.com/equinor/EchoFramework/tree/main/doc/echo-client.md)
+
+---
+
+## Introduction to Echo
+
+The Echo Web project consist a collection of different tools and repositories. Everything a Echo needs to
+communicate with the echo core echo framework and other modules. There are a few repositories associated with
+Echo Web the main repo is EchopediaWeb which to day is the project main code source, All other repo's are
```
Comment (OveBDev) [inline README.md:28]: Web. The
today
all

**Review by OveBDev — APPROVED** [review]

**Pull request merged**